spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/CMul.scala

```diff
@@ -82,7 +82,10 @@ class CMul[T: ClassTag](
   }
 
   private def mulOneDimWeight(dim: Int, expand: Tensor[T], output: Tensor[T]): Unit = {
-    val (innerNum, outerNum) = Utils.getInnerOuterNum(dim, output)
+    val multiplyDimSize = expand.size(dim)
+    var dimOfoutput : Int = 1
+    while(output.size(dimOfoutput) != multiplyDimSize) dimOfoutput += 1
```

Comment:
What if there's no multiplyDimSize in output.size()? TTry to catch the array outofbound error and throw a more meaningful error msg